app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -300,5 +296,15 @@ class BrowserToolbarView(
 
     companion object {
         private const val TOOLBAR_ELEVATION = 16
+
+        @VisibleForTesting
+        internal fun getUrlFromClipboard(store: BrowserStore, customTabSession: Session? = null): String? {
```

Comment:
nit - should this be `getUrlForClipboard`?

Reply:
Oh, yes, definitely. Fixed.